src/main/java/uk/gov/justice/hmpps/prison/service/curfews/OffenderCurfewService.java

```diff
@@ -136,19 +136,19 @@ public List<HomeDetentionCurfew> getBatchLatestHomeDetentionCurfew(final List<Lo
     }
 
     @Transactional
-    @PreAuthorize("hasRole('SYSTEM_USER') and hasAuthority('SCOPE_write')")
+    @PreAuthorize("hasAnyRole('SYSTEM_USER', 'VIEW_PRISONER_DATA') and hasAuthority('SCOPE_write')")
```

Comment:
Wasn't sure whether to add a new CURFEW role or just the VIEW_PRISONER_DATA - went with this as other calls in this service had VIEW_PRISONER_DATA too.